### ISSUE TITLE
多标签分类训练数据问题请教

### ISSUE THREAD

**Issue by xuyao178** [modelscope/ms-swift#3984]
按文档准备了多标签的训练数据，训练会报错。

有2个标签，数据样例：
{"messages": [{"role": "user", "content": "<sentence>"}], "label": [0]}
{"messages": [{"role": "user", "content": "<sentence>"}], "label": [1]}
{"messages": [{"role": "user", "content": "<sentence>"}], "label": [0, 1]}

训练命令：
swift sft \
    --model ./models/Qwen2.5-0.5B \
    --train_type lora \
    --dataset ./train_multi.jsonl \
    --torch_dtype bfloat16 \
    --num_train_epochs 1 \
    --per_device_train_batch_size 16 \
    --per_device_eval_batch_size 16 \
    --learning_rate 1e-4 \
    --lora_rank 8 \
    --lora_alpha 32 \
    --target_modules all-linear \
    --gradient_accumulation_steps 1 \
    --eval_steps 100 \
    --save_steps 100 \
    --save_total_limit 2 \
    --logging_steps 5 \
    --max_length 32768 \
    --output_dir output \
    --warmup_ratio 0.05 \
    --dataloader_num_workers 4 \
    --dataset_num_proc 4 \
    --num_labels 2 \
    --task_type seq_cls \
    --use_chat_template false \
    --problem_type multi_label_classification

报错提示：

![Image](https://github.com/user-attachments/assets/70c4c21a-d9ab-47a3-bdfd-9bb196067a8e)

看报错意思是label里的元素要一致，训练数据是不是要按下面这种方式来提供(标签存在为1，不存在为0)，望大佬指点。
{"messages": [{"role": "user", "content": "<sentence>"}], "label": [0, 1]}
{"messages": [{"role": "user", "content": "<sentence>"}], "label": [1, 0]}
{"messages": [{"role": "user", "content": "<sentence>"}], "label": [0, 1]}

**Comment by Jintao-Huang** [comment]
main分支修复了

**Comment by xuyao178** [comment]
更新到main分支问题解决

**Comment by RyanOvO** [comment]
> 更新到main分支问题解决

哥们，能不能看下你多分类数据集是怎么准备的？官方文档没有找到示例。就只是一个这个，不知道这些1,3,5要怎么对应实际的文本内容，比如1这个label标识xx，3这个label标识xxx。这个对应关联关系是怎么处理的？
{"messages": [{"role": "user", "content": ""}], "label": [1, 3, 5]}
